### DISCUSSION TITLE
Build outputs not detected from a monorepo setup

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#2213] [Help] [answered]
### Summary

I have a monorepo setup on my personal sites in [this repo](https://github.com/kuroji-fusky/website) to host my portfolio and blog together with Turborepo. What should've been working fine typically on their own, Vercel for whatever reason couldn't find the build outputs on both directories.

In the case of my portfolio site written in Nuxt - I've tried overriding the build directories to `.vercel/output` and just `./vercel` that lead up to build output errors. 

Overriding the directory `.nuxt/dist` worked! 

![Screenshot 2023-04-27 232414](https://user-images.githubusercontent.com/94678583/234910162-2e5d0d29-62e3-48bd-badf-ebe8e1fd62a8.png)

...but got a 404 screen when accessing the site, and the site [still currently returns 404](https://kurojifusky.com).

![Screenshot 2023-04-27 225054](https://user-images.githubusercontent.com/94678583/234902249-ed5baf12-604d-4f80-b168-6fd2475101dd.png)

The blog site is working just fine! But pushing a commit has led to the same build output error again.

Usually, this is no problem when deploying the project on its own separately as I initially expected... but I'm not sure what's different in this case assuming my monorepo was set up correctly as I have the root directory turned on both deployments.

**Comment by amyegan** [comment, marked as answer]
I appreciate you sharing the code, @kuroji-fusky! If I try to run on local or deploy the `kuro-landing` code, I get an error about redirects. I suspect you have something installed locally that is missing in the package.json. 

It looks like you're using Turborepo config in the project, so it might also be a good idea to review the [Turborepo Docs](https://turbo.build/repo/docs) to be safe. There's a simplified [Turbo with Yarn example](https://github.com/vercel/turbo/tree/main/examples/with-yarn) repo available for reference and a quickstart guide about [deploying Turborepo to Vercel](https://vercel.com/docs/concepts/monorepos/turborepo#) as well.

**Reply by ghost** [reply]
After skimming through the docs, turns out I haven't specified a build output on both directories, as for `kuro-landing`; I set the build output to its defaults and both deployments succeeded! Thanks for the help!
